server/views/pages/view/cases.njk

```diff
@@ -69,30 +62,40 @@
         }
     ]), licences) %}
 {% endfor %}
-
 {% block content %}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-three-quarters">
             <h1 class="govuk-heading-xl">Select a person to view or print licences and supervision orders</h1>
         </div>
     </div>
-
     <div class="moj-search govuk-!-margin-bottom-6">
         <form method="GET">
             <div class="govuk-form-group">
                 <label class="govuk-visually-hidden" for="search">
                     Search for a case
                 </label>
-                <input class="govuk-input moj-search__input govuk-!-font-size-14" id="search" name="search" placeholder="You can search by name, prison number or probation practitioner" value="{{ search }}">
-            </div>
-            <button class="govuk-button moj-search__button" data-module="govuk-button">
-                Search
-            </button>
-        </form>
-    </div>
-
-    {% if cases.length > 0 %}
-        {{ govukTable({
+                <input class="govuk-input moj-search__input govuk-!-font-size-14" id="search" name="search" placeholder="You can search by name, prison number or probation practitioner" value="{{ search }}"></div>
+                <button class="govuk-button moj-search__button" data-module="govuk-button">
+                    Search
+                </button>
+            </form>
+        </div>
+        {% if hasMultipleCaseloadsInNomis %}
+            {% set comma = joiner() %}
+            <p data-qa="change-caseload">
+                Showing:
+                <span class="govuk-body govuk-!-font-weight-bold" data-qa="caseload-names">
+                    {% for prison in prisonsToDisplay -%}
+                        {{ comma() }}
+                        {{ prison.description }}
+                    {%- endfor %}
```

Comment:
guess you could replace this `joiner` stuff with: 
`prisonsToDisplay  | extractAttr('description') | join(',') `

Reply:
maybe this would be a reason to use it?

Reply:
yep